pkg/cli/admin/catalog/mirrorer_test.go

```diff
@@ -169,11 +169,11 @@ func TestMirror(t *testing.T) {
 					WithTag:    "quay.io/org/coreos-prometheus-operator:76771fef",
 					WithDigest: "quay.io/org/coreos-prometheus-operator@sha256:5037b4e90dbb03ebdefaa547ddf6a1f748c8eeebeedf6b9d9f0913ad662b5731",
 				},
-				"quay.io/test/etcd.0.9.2": {
-					WithTag: "quay.io/org/test-etcd.0.9.2:f0e557b2",
```

Comment:
Was there a reason you opted to update existing tests rather than create new tests?

Reply:
This changes the existing behavior -- mappings from name-only sources get an explicit "latest" tag -- so the affected test cases needed to have their expectations updated.